.github/workflows/dusk_ci.yml

```diff
@@ -22,13 +22,13 @@ jobs:
     name: test core
     uses: dusk-network/.github/.github/workflows/run-tests.yml@main
     with:
-      test_flags: -p phoenix-core --features=alloc
+      test_flags: -p phoenix-core --features=alloc,serde
 
   test_core_rkyv:
     name: test core rkyv compiles
     uses: dusk-network/.github/.github/workflows/run-tests.yml@main
     with:
-      test_flags: -p phoenix-core --features=rkyv-impl,alloc --no-run
+      test_flags: -p phoenix-core --features=rkyv-impl,alloc,serde --no-run
```

Comment:
This test-run just wants to make sure that everything compiles when the `rkyv-impl` feature is enabled.
Since we already know that the code compiles with the `serde` feature enabled (due to the above `test core` test-run) and the `serde` feature has no connection to the `rkyv-impl` feature, I don't think that we need to add the `serde` feature flag to this test-run.